.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: NPM Publish
+
+on:
+  workflow_call:
```

Comment:
[opt] Might be good to also add "workflow_dispatch" just in case you ever need/want to run this manually (e.g. in case of a failure upstream or something.)

Reply:
Nice, yeah good catch.